fontir/src/orchestration.rs

```diff
@@ -69,8 +72,9 @@ macro_rules! context_accessors {
 }
 
 impl Default for Flags {
+    /// Match the way gftools configures fontmake by default
     fn default() -> Self {
-        Flags::MATCH_LEGACY
+        Flags::PREFER_SIMPLE_GLYPHS | Flags::FLATTEN_COMPONENTS
```

Comment:
again, if you ask me, it should not Flags::FLATTEN_COMPONENTS by default

Reply:
I was convinced of this in IM, will update the default.